### DISCUSSION TITLE
Panning when hovering over large node?

### DISCUSSION THREAD

**Discussion by TobiasKrok** [xyflow/xyflow#2495] [Q&A]
Hi!

I have `panOnDrag={true}` which works fine. However, I have some large nodes that can take up the whole viewport at times. How can I switch to panning mode while hovering over nodes without dragging them?

**Comment by moklick** [comment]
Since v11 you can always use middle mouse to pan or you could set `nodesDraggable=false` (then you can pan over them) when a node takes more than 50% of the viewport, for example

**Reply by RAFA3L** [reply]
Hello

I have the same problem, the subflow area use all the screen most of the time and I would like to use a custom handler like the [Drag Handle](https://reactflow.dev/docs/examples/nodes/drag-handle/) and the rest of the subflow area no draggable but paneable. For now I think the solution is using onNodeMouseMove to enable/disable the draggable option when the mouse is over the custom drag handler, is this correct or I missing some option?

Thanks in advance
